libraries/config/src/genesis.cpp

```diff
@@ -63,73 +63,57 @@ void dec_json(Json::Value const& json, Genesis& obj) {
   dec_json(json["dag"], obj.dag);
 }
 
-const Genesis& Genesis::predefined(std::string const& name) {
-  if (auto i = predefined_->find(name); i != predefined_->end()) {
-    return i->second;
-  }
-  throw std::runtime_error("unknown chain config: " + name);
+// const Genesis& Genesis::predefined(std::string const& name) {
```

Comment:
do we want to keep this commented code for some reason ?

Reply:
No, I will remove it. Thought that I removed it already, but I guess it was somehow messed during rebase